.github/workflows/examples-linux-standalone.yaml

```diff
@@ -25,7 +25,7 @@ concurrency:
 jobs:
```

Comment:
We should do this for darwin.yaml as well

Reply:
I will add this to darwin.yaml. Thank you